doc/sphinx/source/extensions.rst

```diff
@@ -2,7 +2,12 @@
 PoCL extensions
 ==================
 
-PoCL currently supports one extension, cl_pocl_content_size.
+PoCL currently supports two extensions, cl_pocl_content_size and cl_khr_command_buffer.
```

Comment:
This seems not to match the above list of host device extensions. This documentation section was originally meant for pocl-specific extensions. Maybe you can just rename the title to 'Extensions' and tell that there are also other extensions supported by the CPU devices, etc.